Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1384,7 +1384,7 @@ extension BrowserViewController: WKNavigationDelegate {
 
         tab.expireSnackbars()
 
-        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) {
+        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) && url.scheme != "about" {
```

Comment:
Is this correct? I thought we completely got rid of `about:` style URLs actually. Did they not get replaced with `http://localhost:*/` style URLs?